src/doc/en/constructions/linear_algebra.rst

```diff
@@ -278,7 +278,7 @@ Another approach is to use the interface with Maxima:
     sage: A = maxima("matrix ([1, -4], [1, -1])")
     sage: eig = A.eigenvectors()
     sage: eig
-    [[[-sqrt(3)*%i,sqrt(3)*%i],[1,1]], [[[1,(sqrt(3)*%i+1)/4]],[[1,-(sqrt(3)*%i-1)/4]]]]
+    [[[-...sqrt(3)*%i...,sqrt(3)*%i],[1,1]], [[[1,(sqrt(3)*%i+1)/4]],[[1,-...(sqrt(3)*%i-1)/4...]]]]
```

Comment:
missed this  one

Reply:
I cannot imagine why we would tell the poor user to use the low-level maxima interface to find eigenvectors in a linear algebra tutorial. My opinion: delete the bad advice (and the failing test).